inc/classes/class-elementary-theme.php

```diff
@@ -48,4 +50,15 @@ public function elementary_theme_support() {
 		// Add support for core block styles.
 		add_theme_support( 'wp-block-styles' );
 	}
+
+	/**
+	 * Block extensions
+	 *
+	 * @since 1.0.0
+	 */
+	public function block_extensions() {
+
+		Media_Text_Interactive::get_instance();
+
+	}
```

Comment:
```suggestion
	public function block_extensions() {
		Media_Text_Interactive::get_instance();
	}
```

Are these empty lines required here? I guess, it's due to the styling in this code?

Reply:
- [x] Removed empty lines